dbcon/joblist/resourcemanager.cpp

```diff
@@ -346,44 +347,44 @@ bool ResourceManager::userPriorityEnabled() const
 // If both have space, return true.
 bool ResourceManager::getMemory(int64_t amount, boost::shared_ptr<int64_t>& sessionLimit, bool patience)
 {
-  bool ret1 = (atomicops::atomicSub(&totalUmMemLimit, amount) >= 0);
+  bool ret1 = (totalUmMemLimit.fetch_sub(amount, std::memory_order_relaxed) >= 0);
```

Comment:
There's no reason to change the common behavior without updating the core tools

Reply:
I didn't pick this up TBH. Can you elaborate on this? What are the core tools?

Reply:
Addressed with a separate commit